build.sbt

```diff
@@ -175,6 +187,7 @@ lazy val commonSettings = instanceSettings ++ clearSourceAndResourceDirectories
         </developer>
       </developers>
   },
+  headerLicense := (ThisBuild / headerLicense).value,
```

Comment:
why is this necessary? (just, out of curiosity)

Reply:
Because I want to define it at ThisBuild, but the plugin doesn't.

Reply:
:)